boards/arm/arduino_portenta_h7/arduino_portenta_h7-common.dtsi

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2022 Benjamin Björnsson <benjamin.bjornsson@gmail.com>.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	gpio_keys {
+		compatible = "gpio-keys";
+		wake_up: button {
+			label = "wakeup";
+			gpios = <&gpioc 13 GPIO_ACTIVE_HIGH>;
+			status = "disabled";
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		red_led: led_0 {
+			gpios = <&gpiok 5 GPIO_ACTIVE_LOW>;
+			label = "Red LED";
+		};
+		green_led: led_1 {
+			gpios = <&gpiok 6 GPIO_ACTIVE_LOW>;
+			label = "Green LED";
+		};
+		blue_led: led_2 {
+			gpios = <&gpiok 7 GPIO_ACTIVE_LOW>;
+			label = "Blue LED";
+		};
+	};
+
+	aliases {
+		led0 = &red_led;
+		led1 = &green_led;
+		led2 = &blue_led;
+	};
+};
+
+&clk_lse {
+	status = "okay";
+};
```

Comment:
Good question, what would make it required?
It should maybe be disabled by default?

Reply:
> Good question, what would make it required?

It would be required if used as a source by an IP (but his is not possible yet).

> It should maybe be disabled by default?

It's already disabled by default in *.dtsi